libs/langchain/pyproject.toml

```diff
@@ -110,6 +110,8 @@ databricks-vectorsearch = {version = "^0.21", optional = true}
 couchbase = {version = "^4.1.9", optional = true}
 dgml-utils = {version = "^0.3.0", optional = true}
 datasets = {version = "^2.15.0", optional = true}
+crate = {version = "^0.34.0", extras=["sqlalchemy"], optional = true}
+cratedb-toolkit = {version = ">=0.0.1", optional = true}
```

Comment:
Better specify `cratedb-toolkit` using an upper version boundary.

Reply:
Downstream version dependencies, for testing 3rd-party software used within the "community" section, are no longer part of LangChain's pyproject.toml files.